blaze-server/src/main/scala/org/http4s/server/blaze/BlazeServer.scala

```diff
@@ -181,6 +185,7 @@ object BlazeBuilder extends BlazeBuilder(
   serviceExecutor = Strategy.DefaultExecutorService,
   idleTimeout = IdleTimeoutSupport.DefaultIdleTimeout,
   isNio2 = false,
+  enableWebSockets = false,
```

Comment:
I'm not certain this shouldn't be on by default.  Seems like one of those annoying flags I forget to flip on a servlet container.